servers/src/mining/stratumserver.rs

```diff
@@ -88,6 +88,11 @@ pub struct JobTemplate {
 	pre_pow: String,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct BlockFound {
```

Comment:
I dont think we really need a BlockFound struct if there will be only a single value.